src/EditorFeatures/TestUtilities/Utilities/TestBackgroundWorkIndicatorService.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Utilities;
+using Moq;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.Utilities
+{
+    internal class TestBackgroundWorkIndicatorService : IBackgroundWorkIndicatorService
```

Comment:
❗ This should not be directly constructed in tests. It should be placed in the `TestComposition` and accessed through MEF.

```suggestion
    [Export(typeof(IBackgroundWorkIndicatorService))]
    [Shared]
    [PartNotDiscoverable]
    internal class TestBackgroundWorkIndicatorService : IBackgroundWorkIndicatorService
```